API/NoDocs/Microsoft.Extensions.Http.Resilience/HttpStandardResilienceOptions.cs

```diff
@@ -0,0 +1,27 @@
+// Assembly 'Microsoft.Extensions.Http.Resilience'
+
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.CompilerServices;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Extensions.Http.Resilience;
+
+public class HttpStandardResilienceOptions
+{
+    [Required]
+    [ValidateObjectMembers]
+    public HttpRateLimiterStrategyOptions RateLimiterOptions { get; set; }
```

Comment:
```suggestion
    public HttpRateLimiterStrategyOptions RateLimiter { get; set; }
```

Do we need the `Options` suffix across these options?